test/metabase/upload_test.clj

```diff
@@ -2265,6 +2292,6 @@
 (defmethod driver/column-name-length-limit ::short-column-test-driver [_] 10)
 
 (deftest unique-long-column-names-test
-  (let [original ["αbcdεf"     "αbcdεfg"   "αbc_2_etc" "αbc_3_xyz"]
+  (let [original ["αbcdεf_αbcdεf"     "αbcdεfg_αbcdεf"   "αbc_2_etc_αbcdεf" "αbc_3_xyz_αbcdεf"]
```

Comment:
Whoops, accidentally spilled in from next change. Needed more characters to test truncation of display names.

No harm leaving it in.